server/postProcessHandler/src/service/dynamodbService.ts

```diff
@@ -0,0 +1,24 @@
+// import { GetItemInput, GetItemOutput } from "aws-sdk/clients/dynamodb";
+
+// import { getItem } from "./dynamodbClient";
+
+// const tableName = process.env.JOB_TABLE as string
+
+// export const addNewJobToDb = async (
+//     email: string,
+//     videoFiles: string[],
+//     imageFiles: string[],
+//   ): Promise<void> => {
+//     const newJobItem = {
+//       TableName: tableName,
+//       Item:{
+//           id: uuidv4,
+//           email,
+//           videoFiles,
+//           imageFiles,
+//           updatedAt: new Date().toISOString()
+//       } 
+//     } as PutItemInput
+  
+//     await putItem(newJobItem)
+//   }
```

Comment:
Except for the imports the commented out code in this file is just a copy of server/uploadHandler/src/service/dynamodbService.ts, so should be removed.

Reply:
Yeah I should delete the commented out code. The reason why I can't import it from `uploadHandler` is explained [here](https://github.com/Visual-Testing-for-Android-Apps/backend/pull/12#discussion_r680648910).